app/components/lg-dob-search-selector.js

```diff
@@ -0,0 +1,25 @@
+/* global moment */
+
+import Ember from 'ember';
+import {dobSearchOptions} from 'lion-guardians/utils/units';
+
+export default Ember.Component.extend({
+  dobSearchOptions: dobSearchOptions,
+  selectedDobOption: dobSearchOptions[0].value,
+
+  selectedSearchDateStart: null,
+  selectedSearchDateEnd: null,
+
+  updateSelectedDates: function() {
+    if (this.get('selectedDobOption')) {
+      var minimumAge = this.get('selectedDobOption.minimumAge'),
+          maximumAge = this.get('selectedDobOption.maximumAge');
+
+      var searchDateStart = moment().subtract(maximumAge, 'years').toDate();
+      var searchDateEnd = moment().subtract(minimumAge, 'years').toDate();
+
+      this.set('selectedSearchDateStart', searchDateStart);
+      this.set('selectedSearchDateEnd', searchDateEnd);
+    }
+  }.observes('selectedDobOption')
```

Comment:
this would probably make more sense to have `selectedSearchDateStart` and end as computed properties, rather than attaching an observer to the selectedDbOption.

Here's what the CP for selectedSearchDateStart would look like:

```
selectedSearchDateStart: function(){
  if (this.get('selectedDobOption')) {
    var maximumAge = this.get('selectedDobOption.minimumAge');
    return moment().subtract(maximumAge, 'years').toDate();
  }
}.property('selectedDobOption')
```